src/selectedtests/app/controllers/health_controller.py

```diff
@@ -0,0 +1,18 @@
+from flask import jsonify
+from flask_restplus import Api, Resource, fields
+
+
+def add_health_endpoints(api: Api):
+
+    ns = api.namespace("health", description="Health check endpoint")
+
+    health_model = ns.model("HealthCheckResponse", {"online": fields.Boolean})
+
+    @ns.route("")
+    class Health(Resource):
```

Comment:
Could you add a docstring to this class?

Reply:
Done